moovingon_ai/assets/oauth_clients.json

```diff
@@ -0,0 +1,25 @@
+{
+  "ui_extension": {
+    "scopes": [],
+    "client_role": "ui_extension"
+  },
+  "integration": {
+    "scopes": [
+      "synthetics_write",
+      "create_webhooks",
+      "api_keys_write",
+      "monitors_write",
+      "synthetics_read",
+      "monitors_read"
+    ],
+    "client_role": "integration",
+    "name": "moovingon.ai",
+    "onboarding_url": "https://demo.moovingon.ai/settings//integrations",
```

Comment:
Is this intentional to include the extra `/` in the `onboarding_url`?

Reply:
@moreauowen Yes.